### PR TITLE
Update interaction when delete data class objects

### PR DESCRIPTION
**Pull request by labkey-susanh**
#### Rationale
Deletion of data class objects now uses the confirmDelete.js script, which displays an Ext4.Msg panel instead of loading a new page. For these tests for error processing, the button text has been updated and we don't expect a page load when there are errors that prevent deletion from happening.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/3623

#### Changes
* Add more parameters to internal method to account for different delete interactions

### CODE REVIEW

File: src/org/labkey/test/tests/TriggerScriptTest.java
Comment (labkey-tchad) [inline src/org/labkey/test/tests/TriggerScriptTest.java]: One second is a little short for a page load. `defaultWaitForPage` is the standard duration.

File: src/org/labkey/test/tests/TriggerScriptTest.java
Comment (labkey-tchad) [inline src/org/labkey/test/tests/TriggerScriptTest.java]: I don't think this needs to check the `expectPageLoad` flag. The surrounding `doAndMaybeWaitForPageToLoad` will handle the page load if necessary. You can pass a zero timeout to `clickButton` to suppress its page load expectation.
```
if (alert != null)
    alert.accept();
else
    clickButton(deleteButtonText, 0);
```

**Review by labkey-tchad — CHANGES_REQUESTED** [review]

**Review by labkey-tchad — APPROVED** [review]
Pre-approving so that you don't have to wait for a re-review.

File: src/org/labkey/test/tests/TriggerScriptTest.java
Comment (labkey-susanh) [reply src/org/labkey/test/tests/TriggerScriptTest.java]: Done.

File: src/org/labkey/test/tests/TriggerScriptTest.java
Comment (labkey-susanh) [reply src/org/labkey/test/tests/TriggerScriptTest.java]: Done.

**Pull request merged**
